eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -17,11 +17,15 @@ TOKEN=$(get_token)
 # set environment vars
 CLOUDSERVER_ENDPOINT="http://${ZENKO_NAME}-connector-s3api.default.svc.cluster.local:80"
 BACKBEAT_API_ENDPOINT="http://${ZENKO_NAME}-management-backbeat-api.default.svc.cluster.local:80"
+VAULT_ENDPOINT="http://${ZENKO_NAME}-connector-vault.default.svc.cluster.local:80"
+ADMIN_ACCESS_KEY_ID=$(kubectl get secret -n zenko data-management-vault-admin-creds.v1 -o jsonpath='{.data.accessKey}' | base64 -d)
+ADMIN_SECRET_ACCESS_KEY$(kubectl get secret -n zenko data-management-vault-admin-creds.v1 -o jsonpath='{.data.secretKey}' | base64 -d)
```

Comment:
@tcarmet not sure what will be the commands here.

Reply:
```suggestion
VAULT_ENDPOINT="http://${ZENKO_NAME}-connector-vault.default.svc.cluster.local:80"
ADMIN_ACCESS_KEY_ID=$(kubectl get secret data-management-vault-admin-creds.v1 -o jsonpath='{.data.accessKey}' | base64 -d)
ADMIN_SECRET_ACCESS_KEY$(kubectl get secret data-management-vault-admin-creds.v1 -o jsonpath='{.data.secretKey}' | base64 -d)
```
I believe the namespace of the secret you are trying to reach needs to match the namespace that zenko was installed. and I think it has been installed in the default namespace

Reply:
How do I find the right names for the entities:
```
++ kubectl get secret data-management-vault-<scality_oci_rw_user>-creds.v1 -o 'jsonpath={.data.accessKey}'
++ base64 -d
Error from server (NotFound): secrets "data-management-vault-<scality_oci_rw_user>-creds.v1" not found
+ ADMIN_ACCESS_KEY_ID=
++ kubectl get secret data-management-vault-<scality_oci_rw_user>-creds.v1 -o 'jsonpath={.data.secretKey}'
++ base64 -d
Error from server (NotFound): secrets "data-management-vault-<scality_oci_rw_user>-creds.v1" not found
```